cypress/integration/pages/liveRadio/testsForAMPOnly.js

```diff
@@ -31,7 +31,7 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
             const { id, externalId } = body.content.blocks[2];
             const serviceId = getMappedServiceId(externalId);
             cy.get(
-              `amp-iframe[src="${`https://www.test.bbc.com/ws/av-embeds/media/${serviceId}/${id}/amp`}"]`,
+              `amp-iframe[src="${envConfig.baseUrl}/ws/av-embeds/media/${serviceId}/${id}/amp"]`,
```

Comment:
Will this fail locally now?

Reply:
```
1) liveRadio - afaanoromoo - Canonical Canonical Tests for afaanoromoo liveRadio live radio body should render an audio player embed:
CypressError: Timed out retrying: Expected to find element: 'iframe[src="http://localhost.bbc.com:7080/ws/av-embeds/media/bbc_afaanoromoo_radio/liveradio"]', but never found it.[0m[90m
```

Reply:
Probably, checking

Reply:
Can add `liveRadioIframeBaseUrl` to `simorgh/cypress/support/config/envs.js`